Makefile

```diff
@@ -143,6 +143,9 @@ ipackage-sim: clean Xcode/ios
 itest: ipackage-sim KIF
 	./scripts/ios/test.sh
 
+ianalyze: clean Xcode/ios
```

Comment:
> I think we should remove it from `ipackage` and `ipackage-sim` too.

Would removing the clean step from these targets have any impact on the release process we have now?